### PR TITLE
Apply changes made on OBOFoundry.github.io

### LINKED ISSUE #140: Sync changes made on OBOFoundry.github.io principles' checks documentation
Many [changes](https://github.com/OBOFoundry/OBOFoundry.github.io/commits/master/principles/checks) were made directly to the pages in OBOFoundry.github.io instead of here. I need to apply the changes here so the update will be in sync.

_Originally posted by @anitacaron in https://github.com/OBOFoundry/OBOFoundry.github.io/issues/2651#issuecomment-2466242102_

### PR DESCRIPTION
**Pull request by anitacaron**
Fixes #140

List of commits from OBOFoundry.github.io applied to this repo:
- https://github.com/OBOFoundry/OBOFoundry.github.io/commit/9c3d971f79d1492e2bc50f1585e3586864bd2ecf
- https://github.com/OBOFoundry/OBOFoundry.github.io/commit/a182e0e57335042c03b0a23f5a6d74db07a96c6b
- https://github.com/OBOFoundry/OBOFoundry.github.io/commit/8fd395bb03c18bc49114b405574b5719ef1539b7
- https://github.com/OBOFoundry/OBOFoundry.github.io/pull/2052/commits/ebc2d0935b3de55076ec61c5a6e4be7cd279c7de

## Release Notes

- **Documentation Enhancements**
	- Improved clarity and consistency across multiple files regarding ontology requirements, including license validation, metadata, and naming conventions.
	- Added detailed instructions for adding licenses, trackers, homepages, and descriptions in ontology metadata.
	- Enhanced comments on error handling and validation logic for better user understanding.
	- Expanded explanations on handling duplicate definitions and labels, emphasizing uniqueness and proper formatting.

- **Error Handling Improvements**
	- Refined error messages for missing entries in ontology metadata, such as trackers and homepages, ensuring clearer communication of issues.
	- Enhanced validation logic for contact information, requiring a single contact person with proper formatting.

### CODE REVIEW

File: util/dashboard/fp_009.py [hunk 1]
```diff
@@ -29,9 +35,11 @@
 ##    - url: http://www.informatics.jax.org/disease/DOID:4123 (link to specific example)
 ##      description: Human genes and mouse homology associated with nail diseases (description of specific example)
 ## ```
+##
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply util/dashboard/fp_009.py:38]: Fixed.

File: util/dashboard/fp_006.py [hunk 2]
```diff
@@ -5,24 +5,29 @@
...
 ##
 ## #### Missing Definitions
+##
 ## Add an [`IAO:0000115` (definition)](http://purl.obolibrary.org/obo/IAO_0000115) annotation to each term that is missing a definition. For help writing good definitions, see [Textual Definitions Recommendations](http://obofoundry.org/principles/fp-006-textual-definitions.html#recommendation).
 ##
-## For adding defintions in bulk, check out [ROBOT template](http://robot.obolibrary.org/template).
+## For adding definitions in bulk, check out [ROBOT template](http://robot.obolibrary.org/template).
 ##
 ## ### Implementation
+##
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply util/dashboard/fp_006.py:30]: Fixed.

File: util/dashboard/fp_002.py [hunk 3]
```diff
@@ -5,12 +5,15 @@
...
+##
 ## 1. Released ontology **must** be in RDF/XML format
 ##
 ## ### Fixes
+##
 ## See the [Common Format Recommendations](http://obofoundry.org/principles/fp-002-format.html#recommendations). [ROBOT](http://robot.obolibrary.org/convert) offers functionality to convert a variety of formats, including OBO, to RDF/XML. Protégé allows you to save ontologies in RDF/XML, as well. The [Ontology 101 Tutorial](https://ontology101tutorial.readthedocs.io/en/latest/StartingProtege.html) has directions on starting and saving in Protégé.
 ##
 ## ### Implementation
+##
 ## Current implementation attempts to load the ontology using OWL API. If the ontology is loaded, it is assumed that it is in a good format, although it may not be RDF/XML. For large ontologies, the ontology is a valid format (either RDF/XML or Turtle) if it can be [loaded with Jena](http://robot.obolibrary.org/query#executing-on-disk) to run the ROBOT report over.
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply util/dashboard/fp_002.py:17]: @coderabitai please create the issue

File: util/dashboard/fp_011.py [hunk 4]
```diff
@@ -19,13 +22,14 @@
...
 ##
 ## ### Implementation
+##
 ## The registry data entry is validated with JSON schema using the [contact schema](https://raw.githubusercontent.com/OBOFoundry/OBOFoundry.github.io/master/util/schema/contact.json). The contact schema ensures that a contact entry is present and that the entry has a name and email address.
 
-import jsonschema
-
 import dash_utils
+import jsonschema
 from dash_utils import format_msg
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply util/dashboard/fp_011.py:30]: Out of scope for this PR.

File: util/dashboard/fp_008.py [hunk 5]
```diff
@@ -5,32 +5,41 @@
...
 ## ```
 ## description: An integrated ontology for the description of life-science and clinical investigations
 ## ```
 ##
 ## ### Implementation
+##
 ## The registry data is checked for 'homepage' and 'description' entries. If either is missing, this is an error. If the homepage is present, the URL is checked to see if it resolves (does not return an HTTP status of greater than 400). If the URL does not resolve, this is also an error.
 
 
 from lib import url_exists
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply util/dashboard/fp_008.py:41]: Out of scope for this PR

File: util/dashboard/fp_003.py [hunk 6]
```diff
@@ -5,29 +5,34 @@
...
 ## ### Implementation
+##
 ## All entity IRIs are retrieved from the ontology, excluding annotation properties. Annotation properties may use hashtags and words due to legacy OBO conversions for subset properties. All other IRIs are checked if they are in the ontology's namespace. If the IRI begins with the ontology namespace, the next character must be an underscore. If not, this is an error. The IRI is also compared to a regex pattern to check if the local ID after the underscore is numeric. If not, this is a warning.
 
-import dash_utils
 import os
 import re
 
+import dash_utils
+
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply util/dashboard/fp_003.py:35]: Out of scope for this PR.

File: util/dashboard/fp_007.py [hunk 7]
```diff
@@ -5,29 +5,33 @@
...
 
 import csv
-import dash_utils
 import os
 import unicodedata
-
 from io import TextIOWrapper
 
+import dash_utils
+
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply util/dashboard/fp_007.py:34]: Out of scope for this PR. Please create an issue.

File: util/dashboard/fp_006.py [hunk 8]
```diff
@@ -5,24 +5,29 @@
 ## Discussion on this check can be [found here](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1010).
 ##
 ## ### Requirements
+##
 ## 1. Each definition **must** be unique.
 ## 2. Each entity **must** not have more than one textual definition.
-## 3. Each entity *should* have a textual definition using [`IAO:0000115` (definition)](http://purl.obolibrary.org/obo/IAO_0000115).
+## 3. Each entity _should_ have a textual definition using [`IAO:0000115` (definition)](http://purl.obolibrary.org/obo/IAO_0000115).
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply util/dashboard/fp_006.py:11]: The change mad is enough.

File: util/dashboard/fp_012.py [hunk 9]
```diff
@@ -5,28 +5,34 @@
...
 ## #### Multiple Labels
+##
 ## Determine which label most accurately describes the term. Change the other label(s) to `oboInOwl:hasExactSynonym` (alternatively, narrow, related, or broad) or [`IAO:0000118` (alternative term)](http://purl.obolibrary.org/obo/IAO_0000118).
 ##
 ## #### Missing Labels
+##
 ## Add an `rdfs:label` annotation to each term that is missing a label. For adding labels in bulk, check out [ROBOT template](http://robot.obolibrary.org/template).
 ##
 ## ### Implementation
+##
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply util/dashboard/fp_012.py:35]: Please create an issue.

File: util/dashboard/fp_012.py [hunk 10]
```diff
@@ -5,28 +5,34 @@
...
 ## Update at least one label to distinguish between the two terms. Add the original label to a `oboInOwl:hasExactSynonym` (alternatively, narrow, related, or broad) or [`IAO:0000118` (alternative term)](http://purl.obolibrary.org/obo/IAO_0000118) annotation.
 ##
 ## If the terms are exactly the same:
+##
 ## 1. Obsolete one of them by adding the `owl:deprecated` annotation property with a boolean value of `true`
 ## 2. Add `obsolete` to the beginning of this term's label
 ## 3. Add a [`IAO:0100001` (term replaced by)](http://purl.obolibrary.org/obo/IAO_0100001) annotation to this term pointing to the other, non-deprecated term.
-##    * Make sure this is an IRI annotation by selecting "IRI Editor" when adding the annotation in Protégé
+##    - Make sure this is an IRI annotation by selecting "IRI Editor" when adding the annotation in Protégé
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply util/dashboard/fp_012.py:24]: Not necessary.

File: util/dashboard/fp_008.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (anitacaron) [reply util/dashboard/fp_008.py:41]: Please create an issue.

File: util/dashboard/fp_005.py [hunk 11]
```diff
@@ -5,15 +5,19 @@
...
 ## ### Fixes
+##
 ## First, read the [FAQ](http://obofoundry.github.io/faq/how-do-i-edit-metadata.html) on how to edit the metadata for your ontology. Then, add the following to your [metadata file](https://github.com/OBOFoundry/OBOFoundry.github.io/tree/master/ontology) (replacing with your domain):
+##
 ## ```
 ## domain: experiments
 ## ```
 ##
 ## ### Implementation
+##
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply util/dashboard/fp_005.py:20]: Please create an issue.

File: util/dashboard/fp_009.py [hunk 12]
```diff
@@ -29,9 +35,11 @@
...
 ## You may have multiple exampels for each user, and mulitple users under the `usages` tag.
 ##
 ## ### Implementation
+##
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply util/dashboard/fp_009.py:42]: Fixed.

File: util/dashboard/fp_008.py [hunk 13]
```diff
@@ -5,32 +5,41 @@
...
 ##
 ## #### Adding a Description
+##
 ## Add the following to your [metadata file](https://github.com/OBOFoundry/OBOFoundry.github.io/tree/master/ontology) (replacing with a short description about your ontology):
+##
 ## ```
 ## description: An integrated ontology for the description of life-science and clinical investigations
 ## ```
 ##
 ## ### Implementation
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply util/dashboard/fp_008.py:35]: Not necessary.

**Review by matentzn — APPROVED** [review]
